Documentation.md

```diff
@@ -347,7 +347,7 @@ Generate a custom pay link
 ```js
 const custom = await client.generatePayLink({
  "title" : "my custom checkout",
- "custom_message" : "some custom message"
+ "custom_message" : "some custom message",
```

Comment:
This documentation is generated from JSDoc tags in the `sdk.js`. Please update that first and regenerate the documentation.